app/src/main/res/xml/grayscale_icon_map.xml

```diff
@@ -631,7 +634,7 @@
     <icon drawable="@drawable/openapk" package="com.dkanada.openapk" name="OperaAPK" />
     <icon drawable="@drawable/openfoodfacts" package="openfoodfacts.github.scrachx.openfood" name="OpenFoodFacts" />
     <icon drawable="@drawable/openkeychain" package="org.sufficientlysecure.keychain" name="OpenKeychain" />
-    <icon drawable="@drawable/opentracks" package="de.dennisguse.opentracks" name="OpenTracks" />
```

Comment:
Was this line deleted intentionally? Could you add it back please?

Reply:
Yeah, that was by mistake... I'll add it back.

Reply:
To which size exactly?